### PR TITLE
Add depth map image log

### PR DESCRIPTION
**Pull request by melisandeteng**
- log depth maps images in comet figure
- fix depth map conversion from unity dataset
- change interpolation to nearest mode for depth maps

- Need *help* to find out why the same generated mask is displayed from all images of the same domain (real / sim) in an image log in comet. 
You can see an example for that problem in [this comet experiment](https://www.comet.ml/melisandeteng/floodmasksdepth/40724aa5532c4fb283a6900e5fdd36b9?experiment-tab=images)

### CODE REVIEW

**Comment by vict0rsch** [comment]
You really need to find out what's happening with you commits, it's hard to review :(

File: data/transforms.py [hunk 1]
```diff
@@ -1,130 +1,156 @@
...
-
-        conf_transforms.append(get_transform(t))
-
-    return conf_transforms + last_transforms
+import torch
+from torchvision import transforms as trsfs
+import torchvision.transforms.functional as TF
+import numpy as np
+from PIL import Image
+def interpolation(task):
```
Comment (vict0rsch) [inline data/transforms.py:6]: need blank lines, flake8 can't have been happy :)

Comment (melisandeteng) [reply data/transforms.py:6]: I couldn’t use sftp this morning it wouldn’t sync my local changes to the cluster so I had to do all of this without a proper editor ...
